src/compas_rhino/geometry/_geometry.py

```diff
@@ -8,21 +8,26 @@
 from compas.utilities import abstractclassmethod
 
 
-__all__ = ['BaseRhinoGeometry']
-
-
-class BaseRhinoGeometry(object):
-    """Base class for Rhino geometry objects.
+class RhinoGeometry(object):
```

Comment:
would it be required to alias `BaseRhinoGeometry = RhinoGeometry` for backwards compat?